lib/index.js

```diff
@@ -553,4 +554,93 @@ function equals(mat, operand) {
     }
 
     return op1.every((val, ind1) => val.every((ele, ind2) => Math.abs(ele - op2[ind1][ind2]) < 1e-10));
-}   
+}
+
+
+/**
+ * Calculates the adjunct matrix
+ *
+ * @param mat input matrix that initialized the function
+ * @returns adjunct matrix
+ */
+
+ function adjunct(mat) {
```

Comment:
I know it is useful to compute the inverse of a Matrix. I'm already doing that using row elimination. Hence I didn't feel the need to add it. It is quite an expensive operation using adjunct and the complexity increases as the matrix size keeps increasing.

Reply:
Yeah I know that operation is very expensive, i'm sorry ;-;

if you want to add that commit on your lib yet
I'll write unit test and update de readme with a sample

Reply:
if you want I can try to refactoring the function little bit

Reply:
Hey you don't have to apologize. I'm just thinking if it is worth it or not.